forte/data/entry_converter.py

```diff
@@ -0,0 +1,209 @@
+                parent_tid=entry.parent[1],
+                child_pack_id=entry.child[0],
+                child_tid=entry.child[1],
+                tid=entry.tid,
+            )
+        elif isinstance(entry, MultiPackGroup):
+            data_store_ref.add_multipack_group_raw(
+                type_name=entry.entry_type(),
+                member_type=get_full_module_name(entry.MemberType),
+                tid=entry.tid,
+            )
+        elif isinstance(entry, MultiPackGeneric):
+            data_store_ref.add_multipack_generic_raw(
+                type_name=entry.entry_type(),
+                tid=entry.tid,
+            )
+        else:
+            raise ValueError(
+                f"Invalid entry type {type(entry)}. A valid entry "
+                f"should be an instance of Annotation, Link, Group, Generics "
+                "or AudioAnnotation."
+            )
+
+        # Store all the dataclass attributes to DataStore
+        for attribute in entry.__dataclass_fields__:
+            value = getattr(entry, attribute, None)
+            if not value:
+                continue
+            if isinstance(value, Entry):
+                value = value.tid
+            elif isinstance(value, FDict):
+                value = {key: val.tid for key, val in value.items()}
+            elif isinstance(value, FList):
+                value = [val.tid for val in value]
+            data_store_ref.set_attribute(
+                tid=entry.tid, attr_name=attribute, attr_value=value
+            )
+
+        # Cache the stored entry and its tid
+        self._entry_dict[entry.tid] = entry
+
+    def get_entry_object(self, tid: int, pack: PackType) -> EntryType:
+        """
+        Convert a tid to its corresponding entry object.
+        """
+
+        # Check if the tid is cached
+        if tid in self._entry_dict:
+            return self._entry_dict[tid]  # type: ignore
+
+        data_store_ref = pack._data_store  # pylint: disable=protected-access
+        entry_data, entry_type = data_store_ref.get_entry(tid=tid)
+        entry_class = get_class(entry_type)
+        entry: Entry
+        # Here the entry arguments are optional (begin, end, parent, ...) and
+        # the value can be arbitrary since they will all be routed to DataStore.
+        if issubclass(entry_class, (Annotation, AudioAnnotation)):
+            entry = entry_class(pack=pack, begin=0, end=0)
+        elif issubclass(
+            entry_class,
+            (
+                Link,
+                Group,
+                Generics,
+                MultiPackGeneric,
+                MultiPackGroup,
+                MultiPackLink,
+            ),
+        ):
+            entry = entry_class(pack=pack)
+        else:
+            raise ValueError(
+                f"Invalid entry type {type(entry_class)}. A valid entry "
+                f"should be an instance of Annotation, Link, Group, Generics "
+                "or AudioAnnotation."
```

Comment:
So this error message will need to be changed when we support new stuff. Maybe we should store some of these info collectively in one place and then all future development will only need to update that place.

Reply:
I think this exception is still here, probably hard to maintain in the long run, any ideas?

Reply:
This is solved by printing out `SinglePackEntries + MultiPackEntries` instead of listing the entry names one by one.